bindings/python/eblob_python.cpp

```diff
@@ -72,13 +96,18 @@ struct eblob_py_iterator : eblob_iterate_control, boost::python::wrapper<eblob_i
 	}
 
 	static int iterator(struct eblob_disk_control *dc, struct eblob_ram_control *rc __attribute__((unused)),
-			void *data, void *priv, void *thread_priv __attribute__((unused)))
+			    int fd, uint64_t data_offset, void *priv, void *thread_priv __attribute__((unused)))
 	{
+		std::auto_ptr<char> p(new char[dc->data_size]);
```

Comment:
Note that using `std::auto_ptr<T>` is considered a bad practice for quite some time.